sdk/src/main/java/com/oursky/authgear/CustomTabsUIImplementation.kt

```diff
@@ -0,0 +1,68 @@
+package com.oursky.authgear
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.net.Uri
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+import kotlin.coroutines.suspendCoroutine
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class CustomTabsUIImplementation : UIImplementation {
+    override fun openAuthorizationURL(
+        context: Context,
+        options: OpenAuthorizationURLOptions,
+        listener: OnOpenAuthorizationURLListener
+    ) {
+        CoroutineScope(Dispatchers.Default).launch {
```

Comment:
Should we let the user specify the coroutine scope?
Usually we would like to use the scope of the view model / fragement ?

Reply:
I was thinking the same as when I saw this too. But AuthgearCore is using a coroutine scope that the developer cannot modify. I think we want to change this, we want to review the use of coroutine scope of the whole SDK.